### PR TITLE
Use default background color instead of black.

### PR DESCRIPTION
**Pull request by modk**
As a start for issue #4 this patch changes the background color to the (terminal) default instead of black.

### CODE REVIEW

File: nuqql/win.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ def _redraw_win(self):
         self.win.clear()
 
         # color settings on
-        curses.init_pair(1, curses.COLOR_BLUE, curses.COLOR_BLACK)
+        curses.use_default_colors()
```
Comment (hwipl) [inline nuqql/win.py]: Would it be possible to move curses.use_default_colors() to the start() function in ui.py, maybe just above nuqql.config.init_win(stdscr) in line 278?

**Review by hwipl — COMMENTED** [review]
Thanks for the changes! I only have one comment regarding use_default_colors().

File: nuqql/win.py (hunk 1 above)
Comment (modk) [reply nuqql/win.py]: Changed it in the last commit, is it ok like that?

**Review by hwipl — APPROVED** [review]
Looks good to me, thanks :)

**Pull request merged**